src/main/java/com/venafi/vcert/sdk/certificate/PEMCollection.java

```diff
@@ -82,4 +95,67 @@ public static PEMCollection newPemCollection(Certificate certificate, PrivateKey
         }
         return pemCollection;
     }
+
+    public String pemCertificate() {
+        String pem = null;
+        if (!Objects.isNull(this.certificate)) {
+            ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+            try (PemWriter pemWriter = new PemWriter(new OutputStreamWriter(outputStream))) {
+                pemWriter.writeObject(new PemObject("CERTIFICATE", this.certificate.getEncoded()));
+            } catch (CertificateEncodingException e) {
```

Comment:
You can handle both exception here, i.e catch (CertificateEncodingException | IOException e)